pkg/data/domain/cluster/service_fake.go

```diff
@@ -0,0 +1,53 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger/microloggertest"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/giantswarm/kubectl-gs/pkg/data/client"
+)
+
+var _ Interface = &FakeService{}
+
+type FakeService struct {
+	service *Service
+	storage []runtime.Object
+}
+
+func NewFakeService(storage []runtime.Object) *FakeService {
```

Comment:
We'll use this fake service for testing. It uses the official k8s testing client, so resources are actually verified and created locally.